examples/native/index.js

```diff
@@ -1,29 +1,36 @@
-import { AppRegistry, NativeModules } from 'react-native'
-
-import { Database } from '@nozbe/watermelondb'
-import SQLiteAdapter from '@nozbe/watermelondb/adapters/sqlite'
-
-import { mySchema } from './src/models/schema'
-import Blog from './src/models/Blog'
-import Post from './src/models/Post'
-import Comment from './src/models/Comment'
-
-import { createNavigation } from './src/components/helpers/Navigation'
+/**
+ * @format
+ */
+
+import {AppRegistry, LogBox} from 'react-native';
+import App from './App';
+import {name as appName} from './app.json';
+
+import {Database} from '@nozbe/watermelondb';
+import SQLiteAdapter from '@nozbe/watermelondb/adapters/sqlite';
+import {blogSchema} from './src/model/schema';
+import {Post} from './src/model/Post.model.js';
+import {Comment} from './src/model/Comment.model';
+import {Blog} from './src/model/Blog.model';
+
+// The example passes WatermelonDB objects as navigation params, these objects contain functions which triggers this warning.
```

Comment:
Might want to setup an example that uses hooks rather than passing WatermelonDB objects as navigation params like the previous example?

@radex what's your thoughts on this?

Reply:
Why not, I'm neutral on this

Reply:
OK how about followup PR once I look at hooks.